docs/modules/reference/pages/glossary.adoc

```diff
@@ -28,35 +36,54 @@ There, users can find knowledge base articles, support, announcements, and gener
 (See xref:operation:elasticsearch/introduction.adoc#elasticsearch[Elasticsearch Integration].)
 
-Flows:: Summaries of network traffic sent by network devices. 
+Events:: Structured historical records of activities in {page-component-title} and the nodes, interfaces, and services it manages.
+(See xref:operation:events/anatomy-events.adoc[Anatomy of an Event].)
+
+Flows:: Summaries of network traffic sent by network devices.
 (See xref:operation:flows/introduction.adoc[Flows].)
 
 Foreign Source:: A data source hosted outside of your local network.
 To customize monitoring of nodes in a foreign source, you can set a foreign source definition with your desired detectors and policies.
-(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign source definition].)
+(See xref:operation:provisioning/getting-started.adoc#foreign-source-definition[Foreign Source Definition].)
+
+Geolocation:: The actual physical location of a node monitored by {page-component-title}.
+Nodes can be manually assigned a location, or the location can be automatically determined using the xref:reference:provisioning/adapters/geoip.adoc[GeoIP Provisioning Adapter].
+
+Grafana:: Grafana is an open-source analysis and visualization web application.
+It connects to data sources and generates dashboards with charts, graphs, and alerts.
+(See https://grafana.com/[Grafana].)
+OpenNMS HELM is a Grafana plugin that allows users to customize fault management and performance management data from {page-component-title}.
+(See https://docs.opennms.com/helm/latest[HELM Documentation].)
 
 Helm:: Helm is an an OpenNMS application for creating flexible dashboards to display and customize fault management and performance management data from Horizon and Meridian.
 Helm works with the Grafana analytics program to customize the data you can view.
 Not to be confused with the https://helm.sh/[Kubernetes Helm] package manager.
-(See https://docs.opennms.com/helm/latest[HELM documentation].)
+(See https://docs.opennms.com/helm/latest[HELM Documentation].)
 
 Horizon:: Open-source solution from OpenNMS that helps users visualize and monitor everything on their local and remote networks.
 Free, community-driven project includes the latest technology and features delivered through a rapid release cycle.
 
-IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services. 
+IFTTT:: If This Then That (https://ifttt.com/[IFTTT]) is an automation and workflow service that integrates apps, devices, and services.
 {page-component-title} can integrate with IFTTT to trigger actions on other services based on customized alarms.
 (See xref:operation:alarms/ifttt-integration.adoc[IFTTT Integration].)
 
+Instance:: A single, unique copy of a running program.
+If a program has multiple instances, that indicates that the program has been loaded into memory more than once.
+
```

Comment:
"instance" is also used in datacollection for processing some types of data for persisting to disk.

Reply:
I didn't see anything referring to "instance" being used this way in our existing documentation. Would we need to add a clarifying sentence to the entry, or maybe earmark it for future?

Reply:
This is kind of a dictionary definition of the term.  We don't use it to mean anything unique.

Reply:
Okay, so maybe something like this then?

```suggestion
Instance:: A single, unique occurrence of an element, document, or running program.
If a program has multiple instances, that indicates that the program has been loaded into memory more than once.

```

Reply:
I think we should keep it. There were a lot of new people at the time we came up with the list who didn't understand what it meant, particularly in the context of the Cloud Appliance where we specifically create an instance as part of set up. While this glossary isn't associated with the Cloud docs directly, it is a central location where we could link fro the Cloud docs if we wanted to.

We could add Mark's comment "also used in datacollection for processing ..." to the one-sentence definition.

Reply:
```suggestion
Instance:: (1) A single, unique occurrence of an element, document, or running program.
(2) Used in data collection for storing tabular data as separate data points.

```